src/execution/window_executor.cpp

```diff
@@ -794,7 +976,28 @@ void WindowAggregateExecutor::EvaluateInternal(WindowExecutorState &lstate, Vect
 	D_ASSERT(aggregator);
 	auto window_begin = FlatVector::GetData<const idx_t>(lastate.bounds.data[WINDOW_BEGIN]);
 	auto window_end = FlatVector::GetData<const idx_t>(lastate.bounds.data[WINDOW_END]);
-	aggregator->Evaluate(*lastate.aggregator_state, window_begin, window_end, result, count);
+
+	auto &agg_state = *lastate.aggregator_state;
+
+	if (post_exclude_tree) {
```

Comment:
I think this logic should be used into the accelerators themselves. This would also make it clear that custom acceleration has not been implemented.